Simplenote/SPDragBar.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+class SPDragBar: UIView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        configure()
+    }
+    
+    private func configure() {
+        backgroundColor = SPUserInterface.isDark ? .white : .black
```

Comment:
Please take a look at `UIColor+Studio.swift` extension. You can define a pair of light / dark colors and use it here as a single UIColor. The benefit is that it will automatically update when theme changes. 
For example right now if you open an information sheet and change a theme (Simulator -> Features -> Toggle Appearance), the color of the handle won't update.